VPCodecISOMediaFileFormatBinding.md

```diff
@@ -164,6 +164,17 @@ Version 0 is deprecated and should not be used.
 profile MUST be valid for all samples that reference this sample entry, that
 is, profile SHALL be equal to the profile used to encode the sample.
 
+**VP9 Coding Profiles** [^1]
```

Comment:
Other tables don't have a title like this, so defer to you on if you want to be consistent.

Reply:
> Other tables don't have a title like this, so defer to you on if you want to be consistent.

I'll drop it. I added after I thought about VP8, but before I added the note. That's probably enough.